src/components/CustomRequestScreen/Mvx/MVXSignTransactions.vue

```diff
@@ -34,12 +54,7 @@ const currencyStore = useCurrencyStore()
           <span class="w-[120px] capitalize">Gas Fees</span>
           <span
             >{{
-              (
-                (transaction.gasLimit *
-                  transaction.gasPrice *
-                  Math.pow(10, -18)) /
-                currencyStore.currencies['EGLD']
-              ).toFixed(5)
+              calculateGasFeeUSD(transaction, currencyStore.currencies['EGLD'])
```

Comment:
Why not make this a `computed`? What's the purpose of a function if you are not going to re-use it and scope it to this component?